src/human-practices/communication/blog.md

```diff
@@ -0,0 +1,16 @@
+# Blog
+## Overview
+The Simply Synbio initiative from last year is introduced to a new medium of writing. It provides a way to bridge the gap between synthetic biology findings to the general public with the aims to promote discussion and invoke thought into the implications that synthetic biology discoveries brings to our communities. 
+## Context and Scope
+Science knowledge holds an important role in policy and decision making in society, which makes science communication crucial. The Simply Synbio blog is a team-managed blog that writes about new synthetic biology discoveries in a comprehensible format. The science blog is intended for audience members of all backgrounds to be able to understand and take part in the discussion of new breakthroughs with the science community. Along with the written portion of the blog, there is an even further condensed accompaniment posted on Instagram. 
+## Goals and non-goals
+### Goals 
+To communicate science to the general public
+To provide information in an accessible format; in a written form for the general public in mind and through further condensed Instagram posts 
+Practice science communication among members 
+Encourage members to partake in a team-wide activity 
```

Comment:
To make this a list, add `-` before each point:

```
- point 1
- point 2
```

you will get 

- point 1
- point 2